x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/utils.ts

```diff
@@ -19,3 +19,11 @@ export const getStatusColor = (status: number) => {
   if (status >= 500) color = 'danger';
   return color;
 };
+
+export const safeJsonParseAndStringify = (json: string) => {
+  try {
+    return JSON.stringify(JSON.parse(json), null, 2);
+  } catch (e) {
+    return json;
+  }
+};
```

Comment:
FYI I migrated this entirely as-is from App Search, I'm not a super huge fan of the name but I can't really think of a better one? Feel free to shout if you have any ideas 🤷